Subject: [PATCH] 0006-disable-libbacktrace
---
Index: CMakeLists.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/CMakeLists.txt b/CMakeLists.txt
--- a/CMakeLists.txt	(revision ae97779f59a81bc56f804927be57749bb22548fa)
+++ b/CMakeLists.txt	(date 1770371157583)
@@ -65,11 +65,7 @@
 option(IREE_ENABLE_CPUINFO "Enables runtime use of cpuinfo for processor topology detection." ${IREE_ENABLE_CPUINFO_DEFAULT})
 
 # Enable libbacktrace for Linux stack traces by default on Linux.
-if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
-  set(IREE_ENABLE_LIBBACKTRACE_DEFAULT ON)
-else()
-  set(IREE_ENABLE_LIBBACKTRACE_DEFAULT OFF)
-endif()
+set(IREE_ENABLE_LIBBACKTRACE_DEFAULT OFF)
 option(IREE_ENABLE_LIBBACKTRACE "Enables libbacktrace for Linux stack traces." ${IREE_ENABLE_LIBBACKTRACE_DEFAULT})
 
 option(IREE_BUILD_COMPILER "Builds the IREE compiler." ON)
@@ -1145,7 +1141,7 @@
 
 # Build libbacktrace from source for Linux stack traces.
 # Creates an empty target if disabled or not on Linux.
-add_subdirectory(build_tools/third_party/libbacktrace EXCLUDE_FROM_ALL)
+#add_subdirectory(build_tools/third_party/libbacktrace EXCLUDE_FROM_ALL)
 
 if(IREE_BUILD_BINDINGS_TFLITE)
   add_subdirectory(runtime/bindings/tflite)
